carbon-design-system/carbon
# fix(components): update number input styles for focus and hover

Closes #9858

Update the styles for number input to address the conflict between hover and focus in light mode.

#### Changelog

**New**

**Changed**

- Update styles for number input to address a conflict with hover and focus in light mode

**Removed**

#### Testing / Reviewing

- Pull down the PR
- Start the storybook in `packages/react`
- Visit the `NumberInput` story and change the `Default` story to use the `light` prop
- Hover the number input buttons and verify that the hover color is being applied
- Click or focus the number input buttons and verify that the outline is correctly showing around each control
